tests/topotests/bgp_prefix_list_topo1/test_prefix_modify.py

```diff
@@ -0,0 +1,408 @@
+        "r4": {"static_routes": [{"network": IPV4_PF7}, {"network": IPV4_PF6}]}
+    }
+
+    result = verify_rib(tgen, "ipv4", dut, input_dict_route)
+    assert result is True, "Testcase {} : Failed \n Error: {}".format(tc_name, result)
+
+    step("Configure IPv4 and IPv6 prefix-list")
+    input_dict_3 = {
+        "r3": {
+            "prefix_lists": {
+                "ipv4": {
+                    "pf_list_1_ipv4": [
+                        {"seqid": "5", "network": IPV4_PF7, "action": "deny"}
+                    ],
+                }
+            }
+        }
+    }
+    result = create_prefix_lists(tgen, input_dict_3)
+    assert result is True, "Testcase {} : Failed \n Error: {}".format(tc_name, result)
+
+    input_dict_3 = {
+        "r3": {
+            "prefix_lists": {
+                "ipv4": {
+                    "pf_list_1": [
+                        {"seqid": "10", "network": IPV4_PF7, "action": "permit"}
+                    ]
+                }
+            }
+        }
+    }
+    result = create_prefix_lists(tgen, input_dict_3)
+    assert result is True, "Testcase {} : Failed \n Error: {}".format(tc_name, result)
+
+    step(
+        "configure route-map seq to permit IPV4 prefix list and seq"
+        "2 to permit IPV6 prefix list and apply it to out direction on R3"
+    )
+
+    input_dict_3 = {
+        "r3": {
+            "route_maps": {
+                "rmap_match_pf_1": [
+                    {
+                        "action": "permit",
+                        "match": {"ipv4": {"prefix_lists": "pf_list_1"}},
+                    }
+                ]
+            }
+        }
+    }
+    result = create_route_maps(tgen, input_dict_3)
+    assert result is True, "Testcase {} : Failed \n Error: {}".format(tc_name, result)
+
+    input_dict_7 = {
+        "r3": {
+            "bgp": {
+                "address_family": {
+                    "ipv4": {
+                        "unicast": {
+                            "neighbor": {
+                                "r4": {
+                                    "dest_link": {
+                                        "r3": {
+                                            "route_maps": [
+                                                {
+                                                    "name": "rmap_match_pf_1",
+                                                    "direction": "out",
+                                                }
+                                            ]
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    result = create_router_bgp(tgen, topo, input_dict_7)
+    assert result is True, "Testcase {} : Failed \n Error: {}".format(tc_name, result)
+
+    logger.info("waiting for 30 sec to check the route on r4")
+    time.sleep(30)
```

Comment:
`sleep()` is forbidden in tests, it's non-deterministic, please consider using `run_and_expect()` or a loop + check, but not a bare sleep.

Reply:
done.